File: exttool/EmuClient.cs
using System;
using System.Collections.Generic;
using System.Security.Cryptography.X509Certificates;
using BizHawk.Client.Common;

namespace BirdsEye {
    public class EmuClient{


        private readonly Logging _log;
        public EmuClient(Logging log){
            _log = log;
        }

        public void GetScreenshot(ApiContainer APIs, String path){
            APIs.EmuClient.Screenshot(path);
        }
    }
}

File: birdseyelib/emuClient.py
class EmuClient:
    def __init__(self, client) -> None:
        self.client = client
        self.path = ""

    def setPath(self, path):
        self.path = path
    
    def requestScreenshot(self, path = None):
        if path:
            self.path = path
        
        self.client._queue_request("SCREENSHOT;" + self.path + "\n")

    def LoadState(self, name):
        self.client._queue_request("LOADSTATE;" + name +"\n")